index.js

```diff
@@ -70,6 +70,38 @@ Builder.prototype.build = function(moduleName, outFile, opts) {
   });
 };
 
+var operatorFunctions = {
+  '+': builder.addTrees,
+  '-': builder.subtractTrees
+};
+
+Builder.buildExpression = function(firstModule, operations, cfg) {
```

Comment:
Sorry for stunted communication, net keeps dropping here. Yeah I think moving the parse up makes sense. Can imagine gulp chain building or config files containing strings.